<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calc</class>
 <widget class="QMainWindow" name="calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>684</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>calc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Display">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: gray;
	border: 1px solid gray;
	color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="shortcut">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupNumbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="shortcut">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupNumbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="shortcut">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupNumbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Buttonsin">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttondiv">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="shortcut">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupNumbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="shortcut">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupNumbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Buttonmul">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="shortcut">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="shortcut">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupNumbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Buttoncos">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupNumbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupNumbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Buttonadd">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="shortcut">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupNumbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Buttontan">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonAC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPoint">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="shortcut">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonsub">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="shortcut">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="shortcut">
     <string>0</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroupNumbers</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Buttoneq">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonln">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonopen">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
    <property name="shortcut">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonclose">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
    <property name="shortcut">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonpow">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>pow</string>
    </property>
    <property name="shortcut">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonx">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="shortcut">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonsign">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonmod">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonasin">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonacos">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonatan">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonlog">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonsqrt">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QWidget" name="frame" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>x from:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>x to:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>y from:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>210</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>y to:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupNumbers">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
